src/plugins/data_source_management/public/components/custom_database_icon/error_icon.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+
+export const ErrorIcon = () => {
```

Comment:
how's it look in dark mode?

Reply:
This is dark mode 
![Screenshot 2024-04-22 at 9 46 43 AM](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/17714150/95255b89-1840-4743-b4a5-1ceb9607f017)